### PR TITLE
1 Provide automated scripts for benchmarking and visualizing results

### PR DESCRIPTION
**Pull request by mknaranja**
In order to benchmark our solver (for the upcoming complexity paper), we should provide some automated scripts for executing GMGPolar, collecting the results and plotting them.

### CODE REVIEW

File: performance/run_gmgpolar.sh [hunk 1]
```diff
@@ -1,39 +1,50 @@
...
 
-echo "#SBATCH -N 1" >> run_gmgpolar_sbatch.sh
-echo "#SBATCH -n 1" >> run_gmgpolar_sbatch.sh
-echo "#SBATCH -c 14" >> run_gmgpolar_sbatch.sh
+echo "#SBATCH -N $nodes" >> run_gmgpolar_sbatch.sh
+echo "#SBATCH -n $ranks" >> run_gmgpolar_sbatch.sh
+echo "#SBATCH -c $cores" >> run_gmgpolar_sbatch.sh
 echo "#SBATCH -t 6000" >> run_gmgpolar_sbatch.sh
-echo "#SBATCH --exclusive" >> run_gmgpolar_sbatch.sh
+# echo "#SBATCH --exclusive" >> run_gmgpolar_sbatch.sh
```
Comment (mknaranja) [inline performance/run_gmgpolar.sh:47]: reactivate. development artifact.

**Review by carolakruse — APPROVED** [review]
Seems all ok to me.

File: performance/run_gmgpolar.sh [hunk 2]
```diff
@@ -1,58 +1,129 @@
...
-mod_pk=0		# mod_pk (0/1)
-R0=0.1			# r (1e-8/1e-5/1e-2)
-DirBC_Interior=0	#  DirBC_Interior (0/1)
-divideBy2=0		# divideBy2 (3/4/5/6)
-smoother=3		# smoother (3,13)
-extrapolation=0		# E
+mod_pk=1		    # mod_pk
+prob=5			    # prob
+# TODO: which alpha and beta to simulate?
+alpha_coeff=2
```
Comment (mknaranja) [inline performance/run_gmgpolar.sh:36]: We need to verify that the alpha descent is aligned with the grid!

File: performance/run_gmgpolar.sh [hunk 3]
```diff
@@ -1,58 +1,129 @@
...
 
 #changing variables
-mod_pk=0		# mod_pk (0/1)
-R0=0.1			# r (1e-8/1e-5/1e-2)
-DirBC_Interior=0	#  DirBC_Interior (0/1)
-divideBy2=0		# divideBy2 (3/4/5/6)
-smoother=3		# smoother (3,13)
-extrapolation=0		# E
+mod_pk=1		    # mod_pk
+prob=5			    # prob
```
Comment (mknaranja) [inline performance/run_gmgpolar.sh:34]: CartesianR6GyroSonnendruckerCircular

**Review by Philou31 — APPROVED** [review]
Looks good to me.

I admit that I looked only in detail the first files, and not the ones generated by Emily.

**Pull request merged**